packages/code-snippet/src/CodeSnippetService.ts

```diff
@@ -64,7 +64,7 @@ export class CodeSnippetService {
     return codeSnippetsByLanguage;
   }
 
-  static deleteCodeSnippet(codeSnippet: ICodeSnippet): Promise<void> {
+  static deleteCodeSnippet(codeSnippet: ICodeSnippet): Promise<boolean> {
```

Comment:
I think it may also be worth adding a jsdoc comment for this function that clearly state the meaning of the boolean return